src/app/transactions/get-transactions-metadata-by-ids.ts

```diff
@@ -0,0 +1,6 @@
+import { TransactionsMetadataRepository } from "@services/ledger/services"
```

Comment:
is this not part of wallet folder?

Reply:
It can be in either `wallet` or `account` because we have transactions use-case in both. I wasn't sure where to put these given that 🤔